features/base/test/rkhunter.d/rkhunter.conf

```
@@ -13,6 +13,7 @@ ENABLE_TESTS=ALL
 DISABLE_TESTS=immutable suspscan hidden_ports hidden_procs deleted_files packet_cap_apps apps
 ALLOWDEVFILE=/dev/*
 ALLOWDEVFILE=/dev/shm/*
+ALLOWDEVFILE=/dev/shm/jack_db/*
```

Shouldn't this be more like
```suggestion
ALLOWDEVFILE=/dev/shm/*/*
```

or, if it uses a proper[tm] shell
```suggestion
ALLOWDEVFILE=/dev/shm/**/*
```

---

When it comes to whitelisting I'll always prefer an explicit definition. To be fair this would allow everything within 'jack_db'. However, this is '/dev/shm' and shouldn't matter in this case.

---

If `jack` would have been used in the build, sure. But in this case, those are files from the surrounding system and opaque to our build stuff.

---

ok, legit. Changed it as requested.